modules/infinite-scroll/infinity.php

```diff
@@ -1538,9 +1538,15 @@ function footer() {
 	 *
 	 * @uses __, wp_get_theme, apply_filters, home_url, esc_attr, get_bloginfo, bloginfo
 	 * @return string
+	 *
 	 */
 	private function default_footer() {
-		$credits = sprintf(
+		if ( get_privacy_policy_url() !== '' ) {
```

Comment:
Should we check if `get_privacy_policy_url()` is defined (Only WP 4.9.6 and later define it ) ?